joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/utils/dependency/DependencyResolver.scala

```diff
@@ -84,12 +83,14 @@ object DependencyResolver {
     projectDir: Path
   ): Option[collection.Seq[String]] = {
     logger.info("resolving Gradle dependencies at {}", projectDir)
-    val gradleProjectName = params.forGradle.getOrElse(GradleConfigKeys.ProjectName, defaultGradleProjectName)
-    val gradleConfiguration =
-      params.forGradle.getOrElse(GradleConfigKeys.ConfigurationName, defaultGradleConfigurationName)
+    val gradleProjectName   = params.forGradle.get(GradleConfigKeys.ProjectName)
```

Comment:
Are you sure that does not throw if the key is not found?
Maybe we should make this mor fault tolerant here.

Reply:
`params.forGradle` is a scala Map, so the `get` call returns an option. I'll rename the variables to make this clearer.